pkgs/games/jazz2/default.nix

```diff
@@ -0,0 +1,46 @@
+{ cmake
+, fetchFromGitHub
+, glew
+, glfw
+, lib
+, libGL
+, libopenmpt
+, libvorbis
+, openal
+, SDL2
+, stdenv
+, xorg
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "jazz2";
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    owner = "deathkiller";
+    repo = "jazz2-native";
+    rev = version;
+    sha256 = "sha256-VJqF6fvg3kYVl4zWuo4z4QuDFRGWq5B4N0u/cI7jaKo=";
+  };
+
+  buildInputs = [ libGL SDL2 zlib glew glfw openal libvorbis libopenmpt ];
+  nativeBuildInputs = [ cmake ];
+  propagatedBuildInputs = [ xorg.libSM xorg.libICE xorg.libXext ];
+
+  cmakeFlags = [
+    "-DNCINE_DOWNLOAD_DEPENDENCIES=OFF"
+    "-DGLFW_INCLUDE_DIR=${glfw}/include/GLFW"
+    "-DLIBOPENMPT_INCLUDE_DIR=${libopenmpt.dev}/include/libopenmpt"
+    "-DNCINE_LINUX_PACKAGE=${pname}"
+    "-DNCINE_OVERRIDE_CONTENT_PATH=/run/current-system/sw/share/${pname}/Content"
```

Comment:
That will not work on systems that just use the nix package manager. Not sure what other options we have to solve this.

Reply:
I guess patching the source would be a solution but I doubt in myself that I'll be able to do that.

btw: This is only for autodetecting assets that come with the reimplementation. If it can't detect these files then it will check current working directory, so user can copy assets from /nix/store and let's not forget that they need to obtain and manually install assets from the original game. Check edits on PR because before I contacted the upstream I tried to explain it there.

Reply:
Can you explain what it does, and what needs to be patched in order to fix this (maybe also link to upstream discussion/docs about this)?

I can help with patching if there's no better solution.

Reply:
https://github.com/deathkiller/jazz2-native/blob/master/Sources/Jazz2/ContentResolver.cpp#L104

jazz2 needs not only assets from original game but also has assets for its own features which can be also loaded from current directory when resolving `NCINE_OVERRIDE_CONTENT_PATH` fails